src/Common/src/TypeSystem/Ecma/EcmaType.cs

```diff
@@ -537,6 +537,85 @@ private MarshalAsDescriptor GetMarshalAsDescriptor(FieldDefinition fieldDefiniti
             return null;
         }
 
+        public override PInvokeAttributes GetDelegatePInvokeAttributes()
+        {
+            PInvokeAttributes attributes = PInvokeAttributes.None;
```

Comment:
I think it's confusing to call it PInvokeAttributes given that the term Attribute is overloaded. Perhaps PInvokeFlags?